components/filter/filter.js

```diff
@@ -216,7 +217,7 @@ class Filter extends LocalizeCoreElement(RtlMixin(LitElement)) {
 			return html`<d2l-menu-item text="${dimension.text}" description="${dimensionDescription}">
 				${builtDimension}
 				<div slot="supporting">
-					<span>${this._formatFilterCount(dimension.appliedCount)}</span>
+					<d2l-count-badge number="${dimension.appliedCount}" max-digits=2 hide-zero></d2l-count-badge>
```

Comment:
```suggestion
					<d2l-count-badge number="${dimension.appliedCount}" max-digits="2" hide-zero></d2l-count-badge>
```
Pretty sure it works fine without the quotes in this case, but for consistency we use them everywhere